core/deployment/src/main/java/io/quarkus/deployment/ide/IdeProcessor.java

```diff
@@ -135,7 +139,13 @@ public IdeRunningProcessBuildItem detectRunningIdeProcesses(LaunchModeBuildItem
             return null;
         }
         Set<Ide> result = new HashSet<>(4);
-        List<ProcessInfo> processInfos = ProcessUtil.runningProcesses();
+        List<ProcessInfo> processInfos = Collections.emptyList();
+        try {
+            processInfos = ProcessUtil.runningProcesses();
+        } catch (Exception e) {
+            // this shouldn't be a terminal failure, so just log it to the console
+            log.warn(e);
```

Comment:
Should we log a nice warning message rather than the whole exception? I mean AFAIU it will always be triggered on Apple Silicon?

Reply:
I don't know if it will always be triggered. It looks like a JDK bug honestly.

But yeah, I guess we can just log the message.